### PR TITLE
Corrección typos y sugerencia para "postfijo"

### PR DESCRIPTION
**Pull request by marcorichetta**
Me parece que la palabra sufijo es más utilizada comúnmente.

### CODE REVIEW

File: src/v2/guide/syntax.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ El contenido del `span` se reemplazará con el valor de la propiedad `rawHtml`,
 
 <p class="tip">La ejecución dinámica de HTML arbitrario en su sitio web puede ser muy peligrosa porque puede conducir fácilmente a [vulnerabilidades de XSS](https://en.wikipedia.org/wiki/Cross-site_scripting). Utilice solo la interpolación HTML en contenido de confianza y **nunca** en contenido proporcionado por usuario.</p>
 
-### Attributos
+### Atributos
 
 Los bigotes no se pueden utilizar dentro de los atributos HTML. En su lugar, use una [directiva v-bind](../api/#v-bind):
```
Comment (sxcamacho) [inline src/v2/guide/syntax.md]: Hi @marcorichetta, let me check with the team about the translation for mustaches. Maybe we can change it within your changes. @Tintef Do we have a better translation than `bigotes`?

Comment (sxcamacho) [reply src/v2/guide/syntax.md]: @marcorichetta if you have a suggestion feel free to post it here

Comment (Tintef) [reply src/v2/guide/syntax.md]: I really don't like using `bigotes` to be honest. Let me search for some better examples

Comment (Tintef) [reply src/v2/guide/syntax.md]: @sxcamacho I think we should leave mustaches, and type it in italic as we do with other english words. If necessary, we could add a clarification saying that "mustaches" are `corchetes dobles`.

Here's an example:
```
Los _mustaches_ (doble corchetes) no se pueden utilizar dentro de los atributos HTML.
```

Just like the guys from the [Portuguese translation](https://br.vuejs.org/v2/guide/syntax.html#Texto) did.

What do you think?

Comment (sxcamacho) [reply src/v2/guide/syntax.md]: Much better, I like it, thanks @Tintef 
@marcorichetta please, can you add this change in this PR?

Comment (marcorichetta) [reply src/v2/guide/syntax.md]: Hi! 
The original suggestion I made is about the replacement of `postfijo` with `sufijo`.

The `bigotes` translation it's kind of funny :laughing: and I'm fine with that, but if you're looking for a better translation, why not `llaves`?

Comment (sxcamacho) [reply src/v2/guide/syntax.md]: Yes, it's funny but it isn't a common phrase in Spanish. In this kind of case, we're adding the original word in English and the translation or the idea. I agree with you, `llaves` is correct too. In that case, we could translate this phrase as @Tintef says but using `llaves` or `llaves doble`. 

```suggestion
Los _mustaches_ (llaves doble) no se pueden utilizar dentro de los atributos HTML. En su lugar, use una [directiva v-bind](../api/#v-bind):
```

**Review by sxcamacho — CHANGES_REQUESTED** [review]

**Comment by sxcamacho** [comment]
The original changes requested are okay. Thanks!

File: src/v2/guide/syntax.md (hunk 1 above)
Comment (marcorichetta) [reply src/v2/guide/syntax.md]: Sorry, I replied without @Tintef's answer being load.
I agree and it's also better for consistency between translations.

**Review by sxcamacho — APPROVED** [review]
thanks so much @marcorichetta

**Comment by marcorichetta** [comment]
@sxcamacho Sorry to bother but I realized there are 3 more instances of "bigote". Should I make another PR or could it be done in this one?

**Comment by sxcamacho** [comment]
No problem. Can you create a new one from the same branch?

**Pull request merged**